src/i_joystick.c

```diff
@@ -276,7 +283,8 @@ static int GetButtonsStateGamepad(void)
 
 // Read the state of an axis, inverting if necessary.
 
-static int GetAxisStateGamepad(int axis, int invert, int dead_zone)
+static int GetAxisStateGamepad(int axis, int invert, int dead_zone,
+                               int sensitivity, boolean cubic_response_curve)
```

Comment:
boolean parameter here is bad form. It should be an enum. Both for readability at the call site (making it obvious what it means) and future extensibility.